### ISSUE TITLE
New package: markdown-shortcuts, combining-characters, super-sub-scripts

### ISSUE THREAD

**Issue by jpmvferreira** [federico-terzi/espanso-hub#57]
Repository: https://github.com/jpmvferreira/espanso-mega-pack

To avoid conflicts all triggers are of the form `:trigger:`.

I also have two other packages there: greek-letters and math-symbols, but I know you already have similar packages in the Espanso Hub. My hope to share this packages was that people could gather around into one package to provide all of the required symbols, especially because I took the time to document all of my references and find the best places to get all of the symbols, being a physicist the ones I use are 100% of my use-cases, but there's a few unicode dumps I have of the references that feature 2000+ symbols, so it would be nice to have a hand filtering them and adding them to a common repository! (especially stuff that's harder to find like archaic greek letters that we use once every two years).

**Comment by jpmvferreira** [comment]
I would also like to point out I made a small tool that is able to turn a Espanso .yml package into a Markdown styled table, so that when your package gets big you can easily show in your readme which matches it has, I called it [markpanso](https://github.com/jpmvferreira/markpanso) because I'm not very creative!

**Comment by federico-terzi** [comment]
Hey!

First of all, sorry for the late reply! The new v2 took a long time to complete, so I had to temporarily focus my efforts there :)
Thank you for the help with the package! I've added it to the new [hub repository](https://github.com/espanso/hub), which will be the new standard location for "verified" packages.

A couple of notes:
* This new hub is only accessible through Espanso v2, so if you are using the legacy Espanso version (0.7.3), you 
won't be able to install this package yet. 
After upgrading, you'll be able to install the package with the usual `espanso install <package_name>`
* The hub at [hub.espanso.org](https://hub.espanso.org) is still showing the old repository with the legacy format, so this package is not visible yet.
We are working hard to create a new (and improved!) Hub. Once that will be ready, your package will automatically show there :)
* The v2 uses a slightly different package format, so I converted the package before adding it to the hub.
If you still want to support package installations from the external repo, we might need to commit a couple of updates
to your repo.

The v2 package documentation is not yet there, but I'm working hard to get it up to speed soon :)

Again, thank you for the help, and happy expanding! Feel free to let me know of any concerns you might have

Cheers :)
